<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwoBtnMessageBox</class>
 <widget class="QWidget" name="TwoBtnMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Com_TwoBtnMessageBox_widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QLabel" name="Com_TwoBtnMessageBox_lbMessage">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Com_TwoBtnMessageBox_bt1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Com_TwoBtnMessageBox_bt2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Com_TwoBtnMessageBox_lbEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#E6E6E6;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Com_TwoBtnMessageBox_lbEditname">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>180</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
